src/show.jl

```diff
@@ -83,6 +82,16 @@ for mime in (MIME"text/plain", MIME"text/x-tex", MIME"text/x-latex")
     end
 end
 
+function Base.show(io::IO, ::MIME"text/latex", measure::Complex{<:Measurement})
+    print(io, "\$")
+    Base.show(io, "text/x-latex", measure)
+    print(io, "\$")
+end
+
+function Base.show(io::IO, measure::Complex{<:Measurement})
+    Base.show(io, "text/plain", measure)
+end
```

Comment:
This change looks unrelated to this PR?  It's also untested.

Reply:
Ok, i've added `AbstractFloat` test coverage for this.
It does appear to be an accidental omission,
but if you want i can split this out of the PR?